code/harness/FsSpecific.cpp

```diff
@@ -180,6 +197,10 @@ string XfsFsSpecific::GetFsckCommand(const string &fs_path) {
   return string(kXfsFsckCommand) + fs_path;
 }
 
+string XfsFsSpecific::GetNewUUIDCommand(const string &disk_path) {
+  return string(kXfsFsckCommand) + disk_path;
```

Comment:
Wrong constant to change the UUID of the disk. We are failing to mount because it complains that it has the same UUID :)